Tests/Form/Type/DocumentTypeTest.php

```diff
@@ -112,8 +112,8 @@ public function testManyToManyReferences(): void
         $this->assertInstanceOf(FormView::class, $categoryView);
 
         $this->assertCount(2, $categoryView->children);
-        $this->assertTrue($categoryView->children[0]->vars['checked']);
-        $this->assertFalse($categoryView->children[1]->vars['checked']);
```

Comment:
`children` should be only accessed by string: https://github.com/symfony/symfony/blob/534e1ed986bc0a33e1093f87bb6cede887a23a79/src/Symfony/Component/Form/FormView.php#L34-L39

I tried `"0"` and `"1"`, but it didn't work I guess it because PHP interprets the numeric integer as integer.